test/e2e_node/lifecycle_hooks/prestop_hook_test.go

```diff
@@ -0,0 +1,212 @@
+package lifecycle_hooks
```

Comment:
If it is a new source file, we need to have a license statement at the beginning.